iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/AbstractConnectableXml.java

```diff
@@ -153,31 +151,47 @@ public static void readCurrentLimits(Integer index, Supplier<CurrentLimitsAdder>
                 double value = XmlUtil.readOptionalDoubleAttribute(reader, "value", Double.MAX_VALUE);
                 boolean fictitious = XmlUtil.readOptionalBoolAttribute(reader, "fictitious", false);
                 adder.beginTemporaryLimit()
-                    .setName(name)
-                    .setAcceptableDuration(acceptableDuration)
-                    .setValue(value)
-                    .setFictitious(fictitious)
-                    .endTemporaryLimit();
+                        .setName(name)
+                        .setAcceptableDuration(acceptableDuration)
+                        .setValue(value)
+                        .setFictitious(fictitious)
+                        .endTemporaryLimit();
             }
         });
         adder.add();
     }
 
+    /**
+     * @deprecated Use {@link #writeCurrentLimits(Integer, CurrentLimits, XMLStreamWriter, IidmXmlVersion)} instead.
+     */
+    @Deprecated
```

Comment:
Not sure this should be deprecated: for most people, export network using the current version is OK?